src/index.html

```diff
@@ -17,16 +17,40 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="container">
+      <div
+        class="search-bar big"
+        data-qa="big"
+      >
+        <img
+          class="images big"
+          src="/src/images/Search.svg"
+          alt="search icon"
+        />
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="search-bar__input big"
+        />
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="search-bar small"
+        data-qa="small"
+      >
+        <img
+          class="images small"
+          src="/src/images/Search.svg"
+          alt="search icon"
```

Comment:
The `alt` attribute 'search icon' is an improvement, but consider making it more descriptive by providing context about the image's function, such as 'Search icon for input field'.